Добавил indexOf, lastIndexOf. Разобрался со старыми багами в коде util и aggregate

iv4n-t3a opened the pull request.

Commit:
Add arrows benchmarks

Commit:
Add some column4j benchmarks

Commit:
Add benchmarks result

Commit:
Fix bug with generating column

Commit:
Rename folder with arrows benchmarks to arrows-benchmarks

Commit:
Add tests

Commit:
Add indexOf and lastIndexOf to utils

Commit:
Add index of to aggregate

Commit:
Fix bug in lastIndexOf and lastIndexOfAnother

when to != data.lenght, but loop with vector operation didn't do
anything tail was processed incorectly.

Commit:
Fix bug with tail processing in lastIndexOf and lastIndexOfAnother

Commit:
Update tests for search functions

- Add tests for indexOf and lastIndexOf
- Replace tests for indexOfAnother and lastIndexOfAnother

Commit:
Improve column4j benchmarks

Commit:
Improve arrows benchmarks

Commit:
Remove unused parametr in utils benchmark

Commit:
Fix package declaration in those files

Commit:
Add naive search benchmarking

Commit:
Update benchmarks result

egorlitvinenko commented on analog-benchmarks/src/main/java/org/column4jBench/arrows/AggregatorBenchmark.java:
```diff
@@ -0,0 +1,83 @@
+    @BenchmarkMode(Mode.Throughput)
+    static public void liniarSearchFieldsInitialization(Blackhole blackhole, AggregatorBenchmark benchmark) {
+        blackhole.consume(
+            VectorSearcher.linearSearch(benchmark.vector, new IntComparator(), benchmark.CreateKeyVector(), 0)
```
`benchmark.intcomparator`

Commit:
Add read write and allocation benchmarking for java array

Commit:
Fix mutable column benchmarks

Commit:
Remove useless benchmark

Commit:
Fix naming problems

arrows -> arrow
CreateKeyVector -> createKeyVector
GenerateVector -> generateVector
